Majasdarbs/04-majasdarbs/bootstrap/bootsrap-example.html

```diff
@@ -6,11 +6,20 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="./style-bootstrap.css">
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
 
     <title>Hello, world!</title>
   </head>
   <body>
     <h1>Hello, world!</h1>
+    <div class="container">
+      <div class="row">
+        <div class="col col-xxl-12 col-md-4" id="red"></div>
+        <div class="col col-xxl-12 col-md-4" id="yellow"></div>
+        <div class="col col-xxl-12 col-md-4" id="green"></div>
```

Comment:
another option is to use bootstrap colors https://getbootstrap.com/docs/4.0/utilities/colors/